cmd/profileparser/main.go

```diff
@@ -412,6 +412,10 @@ func main() {
 		ruleName := r.Name
 		// overwrite name
 		r.SetName(getPrefixedName(pb.Name, ruleName))
+		if r.GetLabels() == nil {
+			r.Labels = make(map[string]string)
+		}
+		r.Labels[cmpv1alpha1.RuleIDLabelKey] = ruleName
```

Comment:
So what is the intent with storing the rule in ID? We already have the name stored in `r.Name`, wouldn't an annotation be actually more useful? Or is the expectation that the consumer would translate the ID into the name themselves?

Reply:
The initial intent was to allow indexing, but as we saw with the CO patch, these names could be too long. So I'll move this to an annotation.

Reply:
Done.